draft-ietf-quic-multipath.md

```diff
@@ -112,11 +112,8 @@ non-zero connection IDs in order to identify the path and respective
 packet number space.
 
 To add a new path to an existing QUIC connection with multipath support,
-a client starts a path validation on
+a client starts a path validation or sends a packet on
 the chosen path, as further described in {{path-initiation}}.
-A new path can only be used once the associated 4-tuple has been validated
-by ensuring that the peer is able to receive packets at that address
-(see {{Section 8 of QUIC-TRANSPORT}}).
 In this version of the document, a QUIC server does not initiate the creation
 of a path, but it can validate a new path created by a client.
```

Comment:
I think this is also confusing because there is effectively not really something that explicitly initiates the path.

Reply:
There is: using a connection ID that reference a new path. That's unambiguous, and crypto protected.